specs/default/chef/site-cookbooks/hpcpack/recipes/_get_secrets.rb

```diff
@@ -26,46 +26,31 @@
   action :create
 end
 
+Chef::Log.info( "The managed identity is `#{node[:hpcpack][:identity]}'." )
+
 # Lookup the AD Admin and Cert creds in KeyVault (if present)
 if ! node['hpcpack']['keyvault']['vault_name'].nil?
   Chef::Log.info( "Looking up secrets in vault: #{node['hpcpack']['keyvault']['vault_name']}..." )
 
-
-  if ! node['hpcpack']['keyvault']['admin']['name_key'].nil?
```

Comment:
I suppose admin user name can be passed as plaintext. Do we need to store it in key vault as secret?

Reply:
It's not stored as a secret, so I deleted it.